src/query/storages/fuse/fuse/src/statistics/column_statistic.rs

```diff
@@ -23,6 +26,27 @@ use common_storages_index::SupportedType;
 use common_storages_table_meta::meta::ColumnStatistics;
 use common_storages_table_meta::meta::StatisticsOfColumns;
 
+pub fn calc_column_distinct_of_values(
+    column: &Arc<dyn Column>,
+    column_field: ColumnWithField,
+) -> Result<u64> {
+    let distinct_values = eval_aggr(
+        "approx_count_distinct",
+        vec![],
+        &[column_field],
+        column.len(),
+    )?;
+    if distinct_values.len() > 0 {
```

Comment:
no need to check the len, `distinct_values.len() > 0` is ensured even in empty set.

```
MySQL [(none)]> select max(number) from numbers(3) where number > 333333;
+-------------+
| max(number) |
+-------------+
|        NULL |
+-------------+
1 row in set (0.009 sec)
```